manifest.json

```diff
@@ -5470,6 +5470,10 @@
                         {
                             "url": "https://github.com/Psychpsyo/TNMTMISTYFANLTTSSYCREANKNOWSBIYHWYP/releases/download/1.0.0/TheNeosModThatMakesItSoThatYourFingersAreNoLongerTrustedTouchSourcesSoYouCanRestEasyAtNightKnowingNoOneWillShoveButtonsIntoYourHandWithoutYourPermission.dll",
                             "sha256": "63445a9255dd8e3c3e33cfa99e9bd6878ed5f847e00f6503f957e6e02987553e"
+                        },
```

Comment:
This should be a new version, not an artifact to the existing version

Reply:
Fixed now.